src/IK/include/BipedalLocomotion/IK/QPInverseKinematics.h

```diff
@@ -224,6 +227,7 @@ class QPInverseKinematics : public IntegrationBasedIK
      *                                  5.0, 5.0, 5.0, 5.0, 5.0, 5.0,
      *                                  5.0, 5.0, 5.0, 5.0, 5.0, 5.0)
      * priority                        1
+     * weight_provider_type            ConstantWeightProvider
```

Comment:
I would suggest providing an example of usage in case you want to use something different from ``ConstantWeightProvider``

Reply:
Done c161222f54add61864d296457acca26798b4beb1